processing/index.js

```diff
@@ -0,0 +1,88 @@
+const fs = require('fs')
+const glob = require('glob')
+const rimraf = require('rimraf')
+const program = require('commander')
+const { getAllRevisionData } = require('./modules/find-commit-dates.js')
+const { processData } = require('./modules/process-data.js')
+const { dashboard } = require('./modules/dashboard.js')
+const { splitData } = require('./modules/split-data.js')
+const { optimizeBenchmarks } = require('./modules/optimize-data.js')
+
+const processDataCommand = (options) => {
+  // Load settings
+  const settings = JSON.parse(fs.readFileSync(options.parent.settingsFile))
+
+  // Find all files in the benchmark directory
+  const benchmarkFiles = glob.sync('*.json', {
+    cwd: options.parent.benchmarkResultsDir
+  })
+
+  // Ensure that outdir exists
+  if (!fs.existsSync(options.parent.outDir)) {
```

Comment:
Instead of writing:
```
options.parent
```
All the time, couldn't one pull it out, using variable explosion? Either:
```
const {parent} = options.parent
```
Or:
```
const {outDir} = options.parent
```